packages/cli-main/src/cli/services/kitchen-sink.ts

```diff
@@ -174,5 +182,27 @@ export async function kitchenSink() {
     },
   }
 
+  await renderPrompt({
+    message: 'Associate your project with the org Castile Ventures?',
+    choices: [
+      {label: 'first', value: 'first', key: 'f'},
+      {label: 'second', value: 'second', key: 's'},
+      {label: 'third', value: 'third'},
+      {label: 'fourth', value: 'fourth'},
+      {label: 'fifth', value: 'fifth', group: 'Automations'},
+      {label: 'sixth', value: 'sixth', group: 'Automations'},
+      {label: 'seventh', value: 'seventh'},
+      {label: 'eighth', value: 'eighth', group: 'Merchant Admin'},
+      {label: 'ninth', value: 'ninth', group: 'Merchant Admin'},
+      {label: 'tenth', value: 'tenth'},
+    ],
+    infoTable: {Add: ['new-ext'], Remove: ['integrated-demand-ext', 'order-discount']},
+    onChoose: () => {},
```

Comment:
How is this intended to be used? Is it to subscribe to the selection of an element? If so, what about making it a return element?

```js
const selection = await renderPrompt(...)

// instead of
let selection
await renderPrompt({
  onChoose: (_selection) => _selection = selection
})
```